boards/arm/gd32f450i_eval/support/openocd.cfg

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2021, Teslabs Engineering S.L.
```

Comment:
I'll ask to add ATL-Electronics rights too for this specific file.

Reply:
adjusted